### ISSUE TITLE
Missing labels for all the widget editables

### ISSUE THREAD

**Issue by mateuszzagorski** [ckeditor/ckeditor5#11867]
## :memo: **Provide detailed reproduction steps (if any)**

1.  Open e.g. https://ckeditor.com/docs/ckeditor5/latest/features/table.html#basic-table-features
2.  Launch VoiceOver or other screen reader
3.  Focus the table cell.

### :heavy\_check\_mark: **Expected result**

The editable should be announced as an appropriate element (e.g. table cell).

### :x: **Actual result**

There is no label.

### :question: **Possible solution**

Solution to all the different widget editables should be based on the results from the #11837 issue.

---

If you'd like to see this fixed sooner, add a :+1: reaction to this post.
